src/schemas/fairdata-ida.ui.js

```diff
@@ -100,7 +100,7 @@ export default {
 	'/properties/theme': {
 		'tab': 'description',
 		'title': "Subject heading",
-		'description': "Choose subject headings from the General Finnish Ontology (YSO). It also has English and Swedish translations of the terms.",
+		'description': "Choose subject headings from the KOKO Ontology. It also has English and Swedish translations of the terms.",
```

Comment:
was this change from YSO to KOKO correct?

Reply:
Yes.